analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -236,6 +236,22 @@ def process(self, events):
                 elif region == "4j2b":
                     region_filter = ak.sum(selected_jets.btagCSVV2 > B_TAG_THRESHOLD, axis=1) >= 2
                     selected_jets_region = selected_jets[region_filter]
+                    
+                    pt_var_modifier = (
+                        events[event_filters][region_filter][pt_var]
+                        if "res" not in pt_var
+                        else events[pt_var][jet_filter][event_filters][region_filter]
+                    )
+                    
+                    # overwrite jets object with modified pT (handled by correctionlib in later versions)
+                    selected_jets_region = ak.zip(
+                        {"pt": selected_jets_region.pt * pt_var_modifier,
+                         "eta": selected_jets_region.eta,
+                         "phi": selected_jets_region.phi,
+                         "mass": selected_jets_region.mass,
+                         "btagCSVV2": selected_jets_region.btagCSVV2},
+                        with_name="PtEtaPhiMLorentzVector"
+                    )
 
                     # reconstruct hadronic top as bjj system with largest pT
                     # the jet energy scale / resolution effect is not propagated to this observable at the moment
```

Comment:
This line needs to be removed, as this update takes care of propagating things correctly.